.github/workflows/release.yml

```diff
@@ -6,6 +6,10 @@ on:
       - v*
 
 jobs:
+  repository-dispatch:
```

Comment:
This doesn't seem like a good name for the step/description. 

I'd prefer to use an existing action for this - e.g. https://github.com/marketplace/actions/repository-dispatch

Reply:
Sorted, thanks!👍